app/models/manageiq/providers/openshift/container_manager_mixin.rb

```diff
@@ -32,10 +32,10 @@ def raw_connect(hostname, port, options)
     end
```

Comment:
- The '/oapi' client is a drop in replacement.
- I agree the current logic seems a little confusing and we should be able to improve it when we get the chance.